### PR TITLE
New Minibroker chapter

### LINKED ISSUE #196: Minibroker
Minibroker is a project that allows external services to exist as containers on a Kubernetes cluster, which can be deployed via Helm.

Details on Minibroker including how to run it: https://github.com/osbkit/minibroker

Already working in CAP 1.2.1 is Redis. Details on setup here:
https://trello.com/c/7fHQPF6S/459-5-service-broker-investigation-minibroker

Work on expanding Minibroker support to other services like MariaDB, Mongo, Mysql and Postgres are described here, which is intended for 1.3: https://trello.com/c/HME5FDZw/806-minibroker-for-other-services

Steps for those are mostly identical to Redis though MariaDB will require an additional parameter.

### PR DESCRIPTION
**Pull request by btat**
Resolves #196

### CODE REVIEW

File: xml/cap_admin_minibroker.xml [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.minibroker"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Minibroker</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
```
Comment (cjschroder) [inline xml/cap_admin_minibroker.xml]: Maybe use a more descriptive title, like "Provisioning Services with Minibroker" ?

File: xml/cap_admin_minibroker.xml [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  <link xlink:href="https://github.com/SUSE/minibroker">Minibroker</link> is an
+  <link xlink:href="https://www.openservicebrokerapi.org/">OSBAPI compliant
+  broker</link> created by members of the
+  <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
+  provides a simple method to provision service brokers on Kubernetes clusters.
+ </para>
```
Comment (cjschroder) [inline xml/cap_admin_minibroker.xml]: This looks like a good place for the warning that Minibroker is not yet production-ready, and add some language that it is a technology preview for testing, or some such. "Technology preview" makes it sound cooler.

File: xml/cap_admin_minibroker.xml [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+      <para>
+       <link xlink:href="http://redis.io">Redis</link>
+      </para>
+     </listitem>
+    </itemizedlist>
+    <warning>
+     <title>Do not use Minibroker services on production systems</title>
+     <para>
+      The services used by Minibroker are not production grade and are not recommended for use with production systems.
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_minibroker.xml]: see previous comment

File: xml/cap_admin_minibroker.xml [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+name               url
+minibroker         http://minibroker-minibroker.minibroker.svc.cluster.local
+</screen>
+   </step>
+   <step>
+    <para>
+     Enable access to a service. Services that can be enabled are
+     <literal>mariadb</literal>, <literal>mongodb</literal>,
+     <literal>postgresql</literal>, and <literal>redis</literal>. The example
+     below will uses Redis as the service:
```
Comment (cjschroder) [inline xml/cap_admin_minibroker.xml]: Delete "will"

File: xml/cap_admin_minibroker.xml [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+   This section demonstrates how to use Minibroker services with your
+   applications. The example below uses the Redis service instance created in
+   the previous section.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Obtain the demo application and use <command>cf push</command> with the
+     <literal>--no-start</literal> flag to deploy the application without
```
Comment (cjschroder) [inline xml/cap_admin_minibroker.xml]: I suggest "Obtain the demo application from GitHub"...maybe I'm slow, but it helps me :)

Comment (btat) [reply xml/cap_admin_minibroker.xml]: Updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68

File: xml/cap_admin_minibroker.xml (hunk 4 above)
Comment (btat) [reply xml/cap_admin_minibroker.xml]: Fixed in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68

File: xml/cap_admin_minibroker.xml (hunk 3 above)
Comment (btat) [reply xml/cap_admin_minibroker.xml]: Moved to suggested spot in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68

File: xml/cap_admin_minibroker.xml (hunk 2 above)
Comment (btat) [reply xml/cap_admin_minibroker.xml]: Warning moved to this spot and message updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68

File: xml/cap_admin_minibroker.xml (hunk 1 above)
Comment (btat) [reply xml/cap_admin_minibroker.xml]: Title updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68

**Comment by cjschroder** [comment]
Good stuff, with success and how-to-confirm examples :+1:

**Pull request merged**